components/prism-log.js

```diff
@@ -12,27 +12,31 @@ Prism.languages.log = {
 
 	'level': [
 		{
-			pattern: /\b(?:ALERT|CRIT|CRITICAL|EMERG|EMERGENCY|ERR|ERROR|FAILURE|FATAL|SEVERE)\b/,
+			pattern: /\b(?:ERR|ALERT|CRIT|CRITICAL|EMERG|EMERGENCY|ERR|ERROR|FAILURE|FATAL|SEVERE)\b/,
```

Comment:
We sort word lists alphabetically.

Reply:
I just noticed, it already contains `ERR`. 

This is why these things are sorted.

Reply:
Removed the extra `ERR` and sorted the other terms alphabetically.